inc/container_structs.h

```diff
@@ -14,13 +14,20 @@
 /*******************************************************************************
  * Definitions
  ******************************************************************************/
-/* This structure is used to manage containers timed auto update
+/* This structure is used to manage containers statistic
  * please refer to the documentation
  */
 typedef struct __attribute__((__packed__)) container_stats_t
 {
-    uint8_t msg_fail_ratio;
-    ll_container_stats_t ll_stat;
+    union
+    {
+        struct __attribute__((__packed__))
+        {
+            uint8_t msg_fail_ratio;
+            ll_stats_t *ll_statistic;
+        };
+        uint8_t unmap[sizeof(ll_stats_t)+1]; /*!< streamable form. */
```

Comment:
We should avoid to have pointers in this one because in this case the unmap is useless.
You could pass the pointer of container_stats_t values we want into ll_stat_t instead.